Assets/Scripts/Ammunition.cs

```diff
@@ -2,43 +2,35 @@
 
 public class Ammunition : MonoBehaviour
 {
-    public Transform hand;
+    public Rigidbody body;
     public Vector3 inHandSize = new Vector3(0.2f, 0.2f, 0.2f);
+    public bool canBePickedUp;
 
-    public Vector3 originalScale;
-
+    private Vector3 _originalScale;
     private int _originalLayer;
-    private Rigidbody _body;
 
-    void Start()
+    void Reset()
+    {
+        body = GetComponent<Rigidbody>();
+    }
+
+    public void PickUp()
     {
-        originalScale = transform.localScale;
+        _originalScale = transform.localScale;
 
         var size = GetComponent<MeshFilter>().mesh.bounds.size;
         transform.localScale = new Vector3(inHandSize.x / size.x, inHandSize.y / size.y, inHandSize.z / size.z);
 
-        _body = GetComponent<Rigidbody>();
-        _body.detectCollisions = false;
+        body.detectCollisions = false;
```

Comment:
Might want to make it kinematic as well to disable stuff like gravity, or just manually disabling gravity as well. Or is gravity handled elsewhere?